welltestpy/tools/diagnostic_plots.py

```diff
@@ -0,0 +1,97 @@
+"""
+welltestpy subpackage to make diagnostic plots.
+
+.. currentmodule:: welltestpy.tools.diagnostic_plots
+
+The following classes and functions are provided
+
+.. autosummary::
+   smoothing_derivative
+   plot_diagnostic
+"""
+# pylint: disable=C0103
+import copy
+import warnings
+import functools as ft
+
+import numpy as np
+
+from mpl_toolkits.mplot3d import Axes3D
+import matplotlib.pyplot as plt
+import matplotlib.ticker as ticker
+
+
+
+
+def smoothing_derivative(observation):
```

Comment:
I guess this routine could be in `process`. But I like the idea to have a `tools/diagnostic_plots` submodule.